### DISCUSSION TITLE
Problem on synology NAS

### DISCUSSION THREAD

**Discussion by AlexL1970** [qdm12/gluetun#1548] [Q&A]
Hello everyone. I used Gluetun as I use NordVPN. The configuration was successful immediately and I was able to connect several containers to take advantage of this VPN tunnel. However, I have noticed that the HD every 5 seconds or so, go into read. I say with certainty that it is Gluetun when it is active that creates this "defect" because if I stop this container, the "defect" disappears. Has this happened to anyone? Can you give me some tips to solve this "problem"? Thank you in advance.

**Comment by carmatana** [comment]
May it be the case that one of the containers channeled through gluetun is causing that problem, i.e. your torrent container ?

It may be that when gluetun is down that container has no internet access and so is not doing the write/read thing

Have you tried stopping all the containers using gluetun except it and see if the problem persist?

Just an hypothesis

**Reply by AlexL1970** [reply]
> May it be the case that one of the containers channeled through gluetun is causing that problem, i.e. your torrent container ?
> 
> It may be that when gluetun is down that container has no internet access and so is not doing the write/read thing
> 
> Have you tried stopping all the containers using gluetun except it and see if the problem persist?
> 
> Just an hypothesis

I thought the same thing and as you rightly say, I've tried stopping the associated containers sadly with no luck, the problem persists!

**Comment by carmatana** [comment]
By the way,

Another option that you can try for your VPN is:

https://github.com/binhex/arch-privoxyvpn

I used it for a while with jackett and qbittorrent and it worked very well with PIA

binhex images are excellent

**Reply by AlexL1970** [reply]
> By the way,
> 
> Another option that you can try for your VPN is:
> 
> https://github.com/binhex/arch-privoxyvpn
> 
> I used it for a while with jackett and qbittorrent and it worked very well with PIA
> 
> binhex images are excellent

Thank you for replying and for the tip. I'll try this route, let's see what I can do. Thanks again.
